docs/network-flow-visibility.md

```diff
@@ -754,6 +755,31 @@ Mouse out or click on the background will bring all the traffic back.
 
 <img src="https://downloads.antrea.io/static/08032022/flow-visibility-np-3.png" width="900" alt="Network-Policy Flows Dashboard">
 
+#### Network Topology Dashboard
+
+Network Topology Dashboard visualizes Pod-to-Pod and Pod-to-Service traffic,
+including intra-Node traffic as well. Pods labels are visually grouped by Node.
+Each arrow points from the source to the destination, and is labelled by the
+amount of data transmitted.
+
+<img src="" width="900" alt="Network Topology Dashboard">
```

Comment:
could you fill in the img src?

Reply:
![Screen Shot 2023-02-22 at 6 44 19 PM](https://user-images.githubusercontent.com/10016630/220810363-96de054a-fee4-4e94-8c06-26d204f4c166.png)
![Screen Shot 2023-02-22 at 6 44 51 PM](https://user-images.githubusercontent.com/10016630/220810421-e5db0ad7-35b2-4ba9-a6cc-5a0dee8eaf73.png)
(just uploading here for the URL)